CalibTracker/SiStripHitEfficiency/src/HitEff.cc

```diff
@@ -101,13 +101,25 @@ void HitEff::beginJob(){
   edm::Service<TFileService> fs;
 
   traj = fs->make<TTree>("traj","tree of trajectory positions");
+  #ifdef CALIBTreeDEV
   traj->Branch("TrajGlbX",&TrajGlbX,"TrajGlbX/F");
```

Comment:
TrajGlbX, TrajGlbY, TrajGlbZ should be kept they are used in /CalibTracker/SiStripHitEfficiency/plugins/SiStripHitEffFromCalibTree.cc that reads the tree.

Reply:
@jlagram, OK. Actually I think there has beem a mistake in the list of needed variables in the [survey](https://docs.google.com/spreadsheets/d/1spKxyYckaHmffgHsiJlrgJyadOO6GO4Nz8eKxoobSs8/edit#gid=0). In fact `TrajGlbX`, `TrajGlbY`, `TrajGlbZ` are all marked as unused. I'll correct the spreadsheet.